Tests/FluentAssertions.Equivalency.Specs/DataColumnSpecs.cs

```diff
@@ -78,6 +78,24 @@ public void When_DataColumn_has_changes_but_is_excluded_it_should_succeed()
             .ExcludingColumn(dataColumn2));
     }
 
+    [Fact]
+    public void When_DataColumn_has_changes_but_is_excluded_in_all_tables_it_should_succeed()
```

Comment:
🔧 I suggest to align with the new naming conventions `Can_exclude_a_column_from_all_tables`